src/core/dbus-execute.c

```diff
@@ -1882,6 +1925,67 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "RestrictNamespaces"))
                 return bus_set_transient_namespace_flag(u, name, &c->restrict_namespaces, message, flags, error);
 
+#if HAVE_LIBBPF
```

Comment:
this should not be ifdeffed.

So far we accept all props in all cases, we just don't appy them if we lack the support for them. Hence here too: please parse this stuff, and make it available via the bus props. so that any version of systemd compiled from the same sources accepts and advertises the same set of props with the right data, even if it doesn't actually apply the settings.